### ISSUE TITLE
Need a way to specific different animations for enter vs leave

### ISSUE THREAD

**Issue by chrisnicola** [yearofmoo/ngAnimate-animate.css#5]
There are some problems with using this if you plan to say have a slider (enter from the right, leave to the left) because the animations for these already specify the same side for both enter and leave. I suppose different classes could just be used for these scenarios but I wanted to discuss what might be a better solution.

**Comment by chrisnicola** [comment]
Interestingly up/down don't have this problem because Animate.css treats them as directions of motion so they keep moving that way. Seems like the same should have been done for left/right as well, rather than reversing directions. Perhaps forward/backward would be more apt.

Either way I've made changes in ngAnimate-animate.css to keep these animations moving in one consistent direction when I use them. Happy to submit a PR if it makes sense.

**Comment by matsko** [comment]
Hey @chrisnicola. I've thought about the issue as well, but I haven't made a decision whether or not to use a reverse class or just more animations. Seeing as only up/down/left/right are really only useful, perhaps forward/backward , top-down/bottom-up would be useful. How does this sound to you? Or do you think you could pull something off using a reverse class?

**Comment by chrisnicola** [comment]
Yeah the forward/back top-down/bottom-up or upwards/downwards make sense to me. You could also add a reverse class.
